lib/remap-functions/src/parse_grok.rs

```diff
@@ -103,15 +103,15 @@ impl Expression for ParseGrokFn {
 
                 Ok(Value::from(result))
             }
-            None => Ok(Value::from(BTreeMap::new())),
+            None => Err("unable to parse input with grok pattern".into()),
```

Comment:
I think the error message should be capitalized and end with a period according to the [logging-style](https://github.com/timberio/vector/blob/master/CONTRIBUTING.md#logging-style).

Reply:
So the standards for remap errors are generally lower case without the trailing period since the error raised here is appended to another error message. So that will be reported as: `"function call error: unable to parse input with grok pattern"`, it would look a bit out of sorts if the`U` was capitalized there.

Reply:
That also works, as long we are consistent with it, I guess.